src/api/v1/users/visitors/search.ts

```diff
@@ -40,13 +37,10 @@ const routes: Hapi.ServerRoute[] = [
     handler: async (request: VisitorSearchRequest, h) => {
       const {
         payload: { qrCode },
-        pre: { communityBusiness },
         server: { app: { knex } },
       } = request;
 
-      const visitors = await Visitors.fromCommunityBusiness(knex, communityBusiness);
-
-      const visitor = await findAsync(visitors, (v: User) => compare(qrCode, v.qrCode));
+      const visitor = await Visitors.getOne(knex, { where: { qrCode } });
```

Comment:
Do we want to be able to search across all CBs here? Genuine question.

Reply:
Yes, I guess, as the endpoint is nested under `users`. The app probably wants to hit a cbs/me/users though...

Reply:
I guess let's leave it as is for now.